pallets/file-system/src/utils.rs

```diff
@@ -1117,27 +1117,62 @@ where
         T::Nfts::create_collection(&owner, &owner, &config)
     }
 
-    /// Compute the next block number to insert an expiration after a given TTL.
+    /// Compute the next block number to insert an expiring storage request.
     ///
-    /// This function attempts to insert an expiration item at the next available block after the given TTL starting from
-    /// the current [`NextAvailableExpirationInsertionBlock`].
-    pub(crate) fn queue_expiration_item(
-        expiration_item_ttl: BlockNumberFor<T>,
-        expiration_item: ExpiredItems<T>,
+    /// This function attempts to insert a storage request expiration at the next available block starting from
+    /// the current [`NextAvailableStorageProofExpirationBlock`].
+    pub(crate) fn enqueue_storage_request_expiration(
```

Comment:
I would merge `enqueue_storage_request_expiration` and `enqueue_file_deletion_request_expiration` functions since they do exactly the same thing and have the function accept the expiration item, TTL and next available block. Keeping the `ExpiredItems` enum and implementing 2 functions: `get_ttl` and `get_next_available_expiration_block` which matches on each variant and returns the appropriate values.